metrics/internal/collectors/ocs-client.go

```diff
@@ -35,6 +35,28 @@ func GetOcsClient(opts *options.Options) (*rest.RESTClient, error) {
 	return ocsClient, nil
 }
 
+func GetOcsV1Client(opts *options.Options) (*rest.RESTClient, error) {
+	ocsConfig, err := clientcmd.BuildConfigFromFlags("", opts.KubeconfigPath)
+	if err != nil {
+		return nil, err
+	}
+	scheme := runtime.NewScheme()
+	utilruntime.Must(v1.AddToScheme(scheme))
```

Comment:
```suggestion
	utilruntime.Must(ocsv1.AddToScheme(scheme))
```

Can we rename this to ocsv1?